core/embed/extmod/modtrezorconfig/modtrezorconfig.c

```diff
@@ -212,7 +212,7 @@ STATIC mp_obj_t mod_trezorconfig_set(size_t n_args, const mp_obj_t *args) {
   uint8_t app = trezor_obj_get_uint8(args[0]) & 0x3F;
```

Comment:
I suppose it would make sense to also add a constant for the flags mask 0x3F.

Reply:
Do you think having this as `(0xFF - 0xC0)` is a good (comprehensible) idea?

Reply:
Addressed in 33c9561dc55b362fec4fedc1a69dcbaf85ede705